### DISCUSSION TITLE
rebase router history

### DISCUSSION THREAD

**Discussion by rraammiinn** [vuejs/router#2202] [Help and Questions] [answered]
how I can rebase router history so there be no previous and next routes in history ?

**Comment by posva** [comment, marked as answer]
This is not possible as the browser doesn't allow it: you wouldn't be able to remove entries that come from a different domain. In general, vendors do not want to allow you to "trap" users

**Reply by rraammiinn** [reply]
that's bad.
I wana make it so it feels like a native app.
